Source/WebKit/Scripts/process-entitlements.sh

```diff
@@ -553,6 +553,9 @@ function ios_family_process_webpushd_entitlements()
     plistbuddy Add :com.apple.uikitservices.app.value-access bool YES
     plistbuddy Add :com.apple.private.usernotifications.app-management-domain.proxy string com.apple.WebKit.PushBundles
     plistbuddy Add :com.apple.frontboard.launchapplications bool YES
+    plistbuddy Add :com.apple.private.security.storage.os_eligibility.readonly bool YES
+    plistbuddy Add :com.apple.security.exception.files.absolute-path.read-only array
+    plistbuddy Add :com.apple.security.exception.files.absolute-path.read-only:0 string /private/var/db/os_eligibility/eligibility.plist
```

Comment:
I think it should be sufficient with the sandbox rule for access to `eligibility.plist`.

Reply:
You need the entitlement because the files are in a data vault.

Reply:
Is 'com.apple.security.exception.files.absolute-path.read-only' also required?

Reply:
It was required in my testing. I was a bit surprised that I needed both the sandbox and entitlement change.